src/main/java/teammates/storage/entity/FeedbackSession.java

```diff
@@ -193,6 +192,20 @@ private void adjustResultsVisibleFromTimeIfRequired() {
         }
     }
 
+    @OnLoad
+    @SuppressWarnings("unused") // called by Objectify
+    private void adjustOpeningAndClosingAndVisibleTimeForPrivateSessions() {
+        if (getStartTime() == null) {
+            setStartTime(getCreatedTime());
+        }
+        if (getEndTime() == null) {
+            setEndTime(TimeHelper.parseInstant("2118-01-01 12:00 AM +0000"));
+        }
+        if ("PRIVATE".equals(feedbackSessionType)) {
+            setSessionVisibleFromTime(TimeHelper.parseInstant("2118-01-01 12:00 AM +0000"));
```

Comment:
I realised that the session visible from time is never supposed to be after the session start time:
<img width="940" alt="screen shot 2018-04-04 at 4 58 11 pm" src="https://user-images.githubusercontent.com/6811428/38298251-79f41c2a-3829-11e8-92ea-ad0e54e8fba2.png">

Hence, it is the start time that we need to set to a hundred years later. The visible from time can be set to `TIME_REPRESENTS_FOLLOW_OPENING` so that it follows the start time.